.golangci.yml

```diff
@@ -0,0 +1,205 @@
+version: "2"
```

Comment:
I think copying from stackrox is not the best idea. Let's start with new template with all linters on.